packages/pigeon/lib/objc_generator.dart

```diff
@@ -134,6 +134,85 @@ void _writeClassDeclarations(
   }
 }
 
+String _getCodecName(String? prefix, String className) =>
+    '${_className(prefix, className)}Codec';
+
+String _calcCodecGetterName(String? prefix, String className) =>
```

Comment:
This should be changed as well, as the inconsistency with `_getCodeName` exacerbates my readability concern: the reader is now left to try to understand (in vain, because there's no reason) why two methods that have identical construction use two different verbs.

Reply:
done